### PR TITLE
Project 16 - Isabella Billgren

### PR DESCRIPTION
**Pull request by Bellsibub**
!!See ReadMe for project info & Reflections!!
 
Link to published project: https://inspiring-knuth-358aae.netlify.app/

### CODE REVIEW

File: code/src/components/App/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import store from 'store';
+import Main from 'components/Main';
+import GlobalStyle from './style';
```
Comment (sarianne-peltola) [inline code/src/components/App/index.js:5]: Here also should local and outer dependencies separated?

File: code/src/components/FolderTabs/TabMap/style.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import boxImg from 'assets/mapbox-default.png';
+import boxImgOn from 'assets/mapbox-on.png';
+
+export const Wrapper = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  width: fit-content;
+  margin: 0 auto;
+`;
```
Comment (sarianne-peltola) [inline code/src/components/FolderTabs/TabMap/style.js:11]: maybe extra space here?

**Review by sarianne-peltola — APPROVED** [review]
Very nice project and cool styling! I liked the direction buttons were in a "compass" shape and how they become colored when they are clickable! Nicely also you can follow up where in the path you are. Good job!!

File: code/src/components/ActionCompass/index.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    selected: false,
+    description: null
+  },
+  {
+    type: 'move',
+    direction: 'South',
+    selected: false,
+    description: null
+  }
+];
```
Comment (KarolCislowski) [inline code/src/components/ActionCompass/index.js:31]: Great idea 🤓

**Review by KarolCislowski — APPROVED** [review]
You should be a game dev 😻 I would play a game created by you, for sure 😻

**Pull request closed without merging**